### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by laurensmartina**

### CODE REVIEW

File: system/datatypes/text.php [hunk 1]
```diff
@@ -20,6 +20,7 @@ function __construct($pageListNode, $args) {
 			registerCommandCallback('textRevert', Array($this, 'revert'));
 		}
 
+		/** @deprecated  */
```
Comment (matthijskooijman) [inline system/datatypes/text.php]: This should probably not go into this commit, the commit message does not mention this change. Also, why is this deprecated?

File: system/datatypes/festival.php [hunk 2]
```diff
@@ -231,7 +231,7 @@ function showContributions() {
 		function getSettingsForm() {
 $html = <<<'EOF'
 <table>
-	<tr><td><label for="festival-title">Festival title</label> *</td><td><input name="festival-title"/></td></tr>
+	<tr><td><label for="festival-title">Festival title</label> *</td><td><input id="festival-title" name="festival-title"/></td></tr>
```
Comment (matthijskooijman) [inline system/datatypes/festival.php]: Nitpick: This is a bit of a different change from the rest of the typo fixes, so this might be better placed into its own commit? Not essential, though.

File: system/datatypes/text.php (hunk 1 above)
Comment (laurensmartina) [reply system/datatypes/text.php]: You are right, this should go out, I'll fix it.

File: system/datatypes/festival.php (hunk 2 above)
Comment (laurensmartina) [reply system/datatypes/festival.php]: True, you are right. But let's not do it now, I'll commit more carefully next time.

**Comment by matthijskooijman** [comment]
Looks good! I'd like to see that last commit merged into the first one to clean up the history. You'll need interactive rebase for that. I was going to show you how that works, so this is probably a good example. Let's look at this next tuesday at our meeting?

**Comment by laurensmartina** [comment]
Perfect, let's do that!

**Comment by matthijskooijman** [comment]
Looks perfect now, merging.
